view/frontend/web/js/view/payment/omise-base-method-renderer.js

```diff
@@ -0,0 +1,59 @@
+define(
+    [
+        'Magento_Checkout/js/model/error-processor',
+        'Magento_Checkout/js/model/full-screen-loader'
+    ],
+    function (
+        errorProcessor,
+        fullScreenLoader
+    ) {
+        'use strict';
+
+        return {
+
+            OFFSITE_RETURN_URL: '/orders/:order_id/omise-offsite',
```

Comment:
I cannot remember what it is exactly but it seems not a return url (maybe API Service Provider or something).
@jacstn what name do you think it is? 😂

Reply:
Actually seeing this url is being used only to build a `serviceUrl` and now wonder if it would be better if we just have a function to generate this url? Could be something like 👇 
```js
buildServiceUrl: function (order_id) {
    return urlBuilder.createUrl(
        '/orders/:order_id/omise-offsite',
        { order_id: order_id }
    );
}
```

Although, I am still not sure if we need this – just I forgot what it was for.

Reply:
Can we agree a name here? @jacstn

Then we can move this to a general method in `base`

Reply:
`getMagentoReturnUrl`?

Reply:
Done! Want to check again?